### ISSUE TITLE
[engine] Turn off show error does not work

### ISSUE THREAD

**Issue by alhambrav** [craftercms/craftercms#1064]
I removed the `500.ftl` so i can see the error message instead of the custom error page in the website_editorial bp.  Here are the steps I followed:

1. Edit `crafter-auth-env/bin/apache-tomcat/shared/classes/crafter/engine/extension/server-config.properties` by adding
```
 crafter.engine.template.error.displayInView=true
```
2. Stop and then Start Crafter CMS 
3. Edit one of the page templates to create an error, I modified the article.ftl template 
4. Preview one of the articles, notice the error message on the preview page

![screen shot 2017-07-05 at 3 55 10 pm](https://user-images.githubusercontent.com/25483966/27882361-5fbea5ca-619a-11e7-9e75-1ab0717bd094.png)

**Comment by avasquez614** [comment]
@alhambrav  crafter.engine.template.error.displayInView should be false so that the messages are not displayed. Can you try and let me know?

**Comment by alhambrav** [comment]
@avasquez614 Sure. I see the same behavior for `crafter.engine.template.error.displayInView=true` and `crafter.engine.template.error.displayInView=false`

**Comment by jdrossl** [comment]
@alhambrav the feature is working as expected, if you add a syntax error in a template freemarker will not able to render anything and that is why it is handled as a 500 error. If you add a `${null}` it will simulate a runtime error and you will see a link showing the exception (that can be hidden using the property).

<img width="1097" alt="screen shot 2017-07-10 at 9 45 50 am" src="https://user-images.githubusercontent.com/1236955/28026915-ad85b9d4-6554-11e7-8f23-7942ea47461f.png">

**Comment by alhambrav** [comment]
@jdrossl The feature seems to be working for the home page of the website editorial, like you mentioned in your comment.    

I tried creating an error in the page article template, and I was expecting a link that says **Show error**, like you you have shown in your comment above, but it is displaying the error message like I displayed in the image in the ticket and below.... Is that the expected behavior?

![screen shot 2017-07-10 at 12 41 48 pm](https://user-images.githubusercontent.com/25483966/28029193-37635e78-656d-11e7-8f0a-713c7dd49860.png)

**Comment by alhambrav** [comment]
@jdrossl The stack trace is showing up in the delivery even if I turn off the show error.  Here are the steps I followed, using the website_editorial bp:

1. Stop the server
```
./gradlew stop
```

2.  I edited the `server-config.properties` file and added:
```
crafter.engine.template.error.displayInView=false
```
3. Start again to pick up the changes I made to the `server-config.properties`
```
./gradlew start
```

4.  I create an error in the `home.ftl` file:

![screen shot 2017-07-10 at 4 16 23 pm](https://user-images.githubusercontent.com/25483966/28038016-6b4a0692-658b-11e7-8093-5ffa985f2449.png)

and in the `article.ftl` file:

![screen shot 2017-07-10 at 4 18 53 pm](https://user-images.githubusercontent.com/25483966/28038085-ae341e0c-658b-11e7-8b17-7f7bb6923674.png)

5. I removed the `500.ftl` file.  Delete does not work right now, so Alfonso helped me delete it from the `published`, by using `git cherry-pick`

6. I ran the `init-site.sh` script to setup delivery, then went to the browser to see the delivery:

![screen shot 2017-07-10 at 4 21 49 pm](https://user-images.githubusercontent.com/25483966/28038219-2b087ba8-658c-11e7-9cb0-b3caad3d8904.png)

And here is how an article page looks like:

![screen shot 2017-07-10 at 4 24 25 pm](https://user-images.githubusercontent.com/25483966/28038290-645ad478-658c-11e7-868e-683cdeb85dbc.png)

According to Alfonso, the stack trace should not be displayed if show error is turned off (The "oops ..." part is okay).
